inputs_supply.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: index
+title: Inputs for Modeling Supply
+prev: diagram.html
+next: supply_land.html
+gcam-version: v5.3 
+---
+
+GCAM's supply inputs include information on production, prices, technology cost and performance, and other emissions in the historical period in order to calibrate model parameters. In addition, GCAM's supply modeling requires information on future technology cost and performance and emissions factors for future periods. GCAM requires that supply data is globally consistent with [demand data](inputs_demand.html) for each of its historical model periods as it solves for market equilibrium in these years as it does for future years. These inputs are required for each [region](common_assumptions.html#regional-resolution) and [historical year](common_assumptions.html#historical-years).
+
+# Table of Contents
+
+- [Energy](#energy)
+- [Water](#water)
+- [Food, Feed, and Forestry](#food--feed--forestry)
+
+## External Inputs
+
+### Energy
+
+#### Description
+
+| Name | Description | Type | Source | Resolution | Unit |
+| :--- | :--- | :--- | :--- | :--- | :--- |
+| Historical supply of energy | Supply of energy in the historical period; used for initialization/calibration of GCAM | External data | IEA | Specified by fuel, transformation sector, country, and year |  ktoe and GWh |
+| CO<sub>2</sub> capture rates | Fraction of CO2 captured in CCS technologies. | Assumption |  | Specified by technology and year | unitless |
+| Retirement rules | For vintaged technologies, GCAM requires the user to specify the lifetime, and the parameters required for phased and profit-based shutdown. | Assumption |  | Specified by technology and year | Years (for lifetime), unitless for others |
+| Logit exponents | GCAM requires the user to specify the logit exponents that determine the substitutability between technologies. | Assumption |  | Specified by sector and subsector | N/A |
+| Share weight interpolation rules | These rules dictate how share weights (GCAM's calibration parameter) are specified in future years. | Assumption |  | Specified by sector and subsector | N/A |
```

Comment:
- "Specified by sector and subsector" should be "Specified by subsector and technology".